float32 q
float32 dq
float32 ref_1
float32 ref_2
float32 tau
float32 stiff